include/verilated_vcd_c.cpp

```diff
@@ -362,17 +361,10 @@ void VerilatedVcd::printIndent(int level_change) {
 
 void VerilatedVcd::dumpHeader() {
     printStr("$version Generated by VerilatedVcd $end\n");
-    printStr("$date ");
-    {
-        const time_t tick = time(nullptr);
-        tm ticktm;
-        VL_LOCALTIME_R(&tick, &ticktm);
-        constexpr size_t LEN_BUF = 50;
-        char buf[LEN_BUF];
-        std::strftime(buf, LEN_BUF, "%c", &ticktm);
-        printStr(buf);
-    }
-    printStr(" $end\n");
+    // There used to be a stanza here to emit a $date line.
+    // Suggest just using the file timestamp, instead.
+    // This approach removes redundant functionality and means
+    // artifacts are more likely to be reproducible.
```

Comment:
I did some research and although common I don't see a problem reading files without a date.  I suggest a minor rewording.

```suggestion
    // Verilator used to put in a $date here.  Although $date is shown in
    // IEEE examples, and it is common in VCD writers, VCD readers don't
    // seem to care about it.  Thus, we omit the $date so artifacts are
    // more likely to be reproducible.  However, if $date were to be needed
    // again, SOURCE_DATE_EPOCH should be supported.
```